README.md

```diff
@@ -171,4 +171,33 @@ If a payment method opens in a new browser tab, you must ensure there is a conne
 If the conditions for opening a new tab depend on asynchronous code, the event won’t be trusted and result in an error.
 Refactoring the callback code that opens a new tab in synchronous style can resolve this problem.
 
+You may encounter a similar problem when opening a new window after requesting a token for Pay Station. Below, you can see pseudocode that *WILL NOT* work.
+
+```javascript
+    button.addEventListener("click", async () => {
+      const token = await getToken();
+      window.open(`https://www.pay-station-url-example.com?token=${token}`, "_blank");
+    });
+```
+
+It seems that everything here is logically arranged - first you get the data to open a new window, and then you open it. However, by organizing the code in this way, you lose the connection between the trusted user event and the opening of a new window. 
+
+However, it would be more correct to take a different approach:
+
+1. Immediately open a new window in response to a user-trusted click
+2. Save a reference to the new window
+3. Place a loader in the new window while the main content is not loaded
+4. Send a request to get a token for Pay Station
+5. After receiving the token, form a link for Pay Station taking it into account and pass it to the opened window
```

Comment:
After receiving the token, form a link for Pay Station with this token, and pass it to the opened window.